scripts/parse_tools/parse_checkers.py

```diff
@@ -81,7 +81,7 @@ def check_dimensions(test_val, max_len=0, error=False):
 ########################################################################
 
 # CF_ID is a string representing the regular expression for CF Standard Names
-CF_ID = r"[a-z][a-z0-9_]*"
+CF_ID = r"[A-Za-z][A-Za-z0-9_]*"
```

Comment:
We are never going to try and merge this with the current branch, are we? The current version of `CF_ID` is:
```
CF_ID = r"(?i)[a-z][a-z0-9_]*"
```
which has the same effect but would not merge pretty :)

Reply:
I don't anticipate merging my this with your current code. Rather, your code will overwrite whatever is in the master at that time (that is, the current master plus the changes in this PR - work in progress).

w.r.t. CF_ID, I will just use what you have.